src/services/API/user.js

```diff
@@ -0,0 +1,77 @@
+import { client } from "@/services/API/rootClient.js";
+import {
+  checkUserEndpoint,
+  checkBirthdateEndpoint,
+} from "@/services/API/endpoints.js";
+
+/**
+ * Creates a new student
+ * @param {Object} formData - contains data filled in the form by user
+ */
+export default {
+  studentSignup(formData) {
+    return new Promise((resolve) => {
+      client
+        .post(studentSignupEndpoint, JSON.stringify(formData))
```

Comment:
where is the `studentSingupEndpoint` variable imported? I can see its not imported from the endpoints file

Reply:
I always have trouble with multiple connected PRs. This happens in the `signup` PR.